### PR TITLE
placesVisited: add model,migration,factory,seed,repository.

### PR DESCRIPTION
**Pull request by vladkudinov89**
Pavel,after PR I merge my local project with branch developer & add user model to my task

### CODE REVIEW

File: database/factories/PlacesVisited.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+use Faker\Generator as Faker;
+use Hedonist\Entities\Place\PlacesVisited;
+
+$factory->define(PlacesVisited::class, function (Faker $faker) {
+    return [
+
```
Comment (nemoipaha) [inline database/factories/PlacesVisited.php]: Empty line.

File: database/migrations/2018_08_05_145124_create_places_visiteds_table.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+class CreatePlacesVisitedsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('places_visiteds', function (Blueprint $table) {
```
Comment (nemoipaha) [inline database/migrations/2018_08_05_145124_create_places_visiteds_table.php]: Name can be `visited_places` or `user_place_checkin`.

File: database/seeds/PlacesVisitedTableSeeder.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+
+        for ($i = 1; $i <= 10; $i++)
```
Comment (nemoipaha) [inline database/seeds/PlacesVisitedTableSeeder.php]: Here we should use already existing users and places.

File: app/Entities/Place/PlacesVisited.php [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Hedonist\Entities\Place;
+
+use Illuminate\Database\Eloquent\Model;
+
+class PlacesVisited extends Model
```
[the remark this answers is not in the document]
Comment (fadykstas) [reply app/Entities/Place/PlacesVisited.php]: на Checkin останавливаемся?

File: database/migrations/2018_08_05_145124_create_places_visiteds_table.php (hunk 2 above)
Comment (fadykstas) [reply database/migrations/2018_08_05_145124_create_places_visiteds_table.php]: let's stay on - `user_place_checkin` ?

**Pull request closed without merging**